src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -134,6 +135,7 @@ static void fetch_servers();
 #ifndef DISABLE_HTTP
 static void fetch_servers_callback(http_response_t* response);
 #endif
+static void update_server_list_window();
```

Comment:
Could you replace the times this is called with:
`window_invalidate_by_class(WC_SERVER_LIST);`
and remove `update_server_list_window()`?

Like, I see where it could have been simpler to write out this function for convenience, but it can be achieved by window_invalidate_by_class, without needing another function

Reply:
Oh, ok, I just refactored the existing code. I'm not that familiar with all the window-functions, but sure, that makes more sense.